src/models/ChatMessage.js

```diff
@@ -0,0 +1,44 @@
+const mongoose = require('mongoose');
+
+const chatMessageSchema = new mongoose.Schema(
+  {
+    albumId: {
+      type: String,
+      required: true,
+    },
+    message: {
+      type: String,
+      required: true,
+    },
+    userId: {
+      type: mongoose.Schema.ObjectId,
+      ref: 'User',
+      required: true,
+    },
+    timestamp: {
+      type: Date,
+      default: Date.now,
+    },
+  },
+  {
+    capped: { size: 102400, max: 1000 }, // max 1000 messages
+  }
+);
+
+//console.log('Schema Created');
+
+// Creating a virtual field "album" that references the Album model
+chatMessageSchema.virtual('album', {
+  ref: 'Album', // Reference to the Album model
+  localField: 'albumId', // Local field for the relationship
+  foreignField: 'spotify_id', // Foreign field for the relationship
```

Comment:
Part of the reason that I want to remove this for now, is that there is no `spotify_id` on the Album schema. We _could_ add one and do a backfill to update all the existing documents with a valid value there but I don't think it's worth the effort right now since we don't have a use case fo rit.